pom.xml

```diff
@@ -497,6 +497,12 @@
       <groupId>com.google.gwt.eventbinder</groupId>
       <artifactId>eventbinder</artifactId>
       <version>1.1.0</version>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.gwt</groupId>
+          <artifactId>gwt-user</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
A few of our dependencies transitively pull in `com.google.gwt:gwt-user`. We want them to instead use `org.gwtproject:gwt-user`, so we must manually exclude the `com.google.gwt:gwt-user` dependency.